docs/learn/learn-parachains-protocol.md

```diff
@@ -0,0 +1,107 @@
+---
+id: learn-parachains-protocol
+title: Parachains' Protocol Overview
+sidebar_label: Protocol Overview
+description: High level view of actors and protocols involved in running parachains on Polkadot.
+keywords: [parachains, application-specific, sharding, protocol, validator, collator, nominator]
+slug: ../learn-parachains-protocol
+---
+
+:::info
+
+This page is a summary of the [Protocol Overview chapter in **The Polkadot Parachain Host Implementer's Guide**](https://paritytech.github.io/polkadot/book/protocol-overview.html)
+
+:::
+
+The aim of the protocol is to carry a parachain's block from authoring to inclusion through a process that can be carried out repeatedly and in parallel for each parachain connected to the relay chain.
+
+## Main Actors
+
+### Validators
+
+They are responsible for validating proposed parachain's blocks by checking the **Proof-of-Validity** (PoV) of the blocks and ensuring the PoV remains available. They have "skin in the game", meaning they have funds bonded on-chain that can be partially or fully confiscated in case of misbehavior.
+
+### Collators
+
+They create the PoV that validators know how to check. Creating PoV requires familiarity with transaction format and block authoring rules of a specific parachain, as well as having access to its full state.
+
+### Fishermen
+
+They catch misbehaving validators in exchange for a bounty. Collators and validators can behave as fishermen.
+
+## Summary View
+
+The parachains' protocol can be summarized into three main steps:
+
+1. Collators send to validators parachain blocks with their PoV for check.
+2. Validators validate the block using the PoV, signing statements that can have positive or negative.
+3. With enough positive statements the block can be added to the relay chain.
+
+The figure below shows a representation of a parachain with collators and validators. The figure also shows the journey of a block through the three main steps, as well as the sections where the [Inclusion Pipeline](#inclusion-pipeline) and the [Approval Process](#approval-process) take place (see next Sections).
+
+![parachain-protocol-summary](../assets/parachain-protocol-summary.png)
+
+Negative statements will lead to a dispute, and if there are false negatives whoever will be on the wrong side (once the dispute is resolved) will be slashed. False positives can also happen, those actors responsible for it will also be slashed and whoever reported the case will get a bounty. In order to detect false positives, PoV information must be available after the block has been added to the relay chain so that validators can check the work. PoVs are expected to be between 1 MB and 10 MB; as a consequence they are not included in blocks but they are part of the data availability scheme in which validators must prove that the inputs to their work will remain available.
```

Comment:
Whenever possible, back the statements like these "PoVs are expected to be between 1 MB and 10 MB;" with references

Reply:
yeah, found this in the parity's doc with no ref, maybe should I mention the document?

Reply:
@DrW3RK thanks for the preliminary feedback, I will get back to this PR in late January when I am back. In the meantime, I implemented the Parachain Host Implementer Guide in the current pages of the wiki. I saw there is already a page about "Availability and Validity", I think this page can be merged together with the content in this page here and create a cohesive story about the lifecycle of a parachain block.

Reply:
Yeah, sounds good to me!